src/output-json-file.c

```diff
@@ -149,6 +149,19 @@ json_t *JsonBuildFileInfoRecord(const Packet *p, const File *ff,
     json_object_set_new(fjs, "filename", SCJsonString(s));
     if (s != NULL)
         SCFree(s);
+
+
+    json_t* sig_ids = json_array();
+    if (unlikely(fjs == NULL)) {
+        json_decref(js);
+        return NULL;
+    }
+
+    for (unsigned int i = 0; ff->sid != NULL && i < ff->sid_cnt; i++) {
```

Comment:
s/unsigned int/uint32_t/g to match the explicit type of File::sid_cnt. We prefer explicit types, so uint32_t over unsigned int, uint64_t over unsigned long long, etc.